[2025-03-05T22:34:07.174+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-05T22:34:07.179+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: forecast_stock_price.train_model manual__2025-03-05T22:34:05.995306+00:00 [queued]>
[2025-03-05T22:34:07.181+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: forecast_stock_price.train_model manual__2025-03-05T22:34:05.995306+00:00 [queued]>
[2025-03-05T22:34:07.181+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-03-05T22:34:07.185+0000] {taskinstance.py:2888} INFO - Executing <Task(_PythonDecoratedOperator): train_model> on 2025-03-05 22:34:05.995306+00:00
[2025-03-05T22:34:07.190+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=276) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-05T22:34:07.189+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'forecast_stock_price', 'train_model', 'manual__2025-03-05T22:34:05.995306+00:00', '--job-id', '60', '--raw', '--subdir', 'DAGS_FOLDER/forecast_stock_price.py', '--cfg-path', '/tmp/tmp5ja40tla']
[2025-03-05T22:34:07.191+0000] {standard_task_runner.py:105} INFO - Job 60: Subtask train_model
[2025-03-05T22:34:07.190+0000] {standard_task_runner.py:72} INFO - Started process 279 to run task
[2025-03-05T22:34:07.211+0000] {task_command.py:467} INFO - Running <TaskInstance: forecast_stock_price.train_model manual__2025-03-05T22:34:05.995306+00:00 [running]> on host d0b2b24d510a
[2025-03-05T22:34:07.241+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='forecast_stock_price' AIRFLOW_CTX_TASK_ID='train_model' AIRFLOW_CTX_EXECUTION_DATE='2025-03-05T22:34:05.995306+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-05T22:34:05.995306+00:00'
[2025-03-05T22:34:07.242+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-03-05T22:34:07.243+0000] {connection.py:413} INFO - Snowflake Connector for Python Version: 3.12.1, Python Version: 3.12.5, Platform: Linux-6.10.14-linuxkit-aarch64-with-glibc2.36
[2025-03-05T22:34:07.243+0000] {connection.py:1196} INFO - Connecting to GLOBAL Snowflake domain
[2025-03-05T22:34:07.243+0000] {connection.py:1277} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2025-03-05T22:34:07.843+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T22:35:51.829+0000] {cursor.py:1156} INFO - Number of results in first chunk: 1
[2025-03-05T22:35:53.546+0000] {cursor.py:1156} INFO - Number of results in first chunk: 14
[2025-03-05T22:35:53.548+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-03-05T22:35:53.575+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-03-05T22:35:53.576+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=forecast_stock_price, task_id=train_model, run_id=manual__2025-03-05T22:34:05.995306+00:00, execution_date=20250305T223405, start_date=20250305T223407, end_date=20250305T223553
[2025-03-05T22:35:53.632+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-05T22:35:53.682+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-05T22:35:53.684+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
